ecommerce/migrations/0038_product_is_private.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.2.20 on 2023-08-29 12:38
+
+from django.db import migrations, models
+
+
+def populate_is_private(apps, schema_editor):
+    """
+    Inverts and Migrates `Product.visible_in_bulk_form` field data to `Product.is_private`
+    as we will remove `Product.visible_in_bulk_form`.
+    """
+    Product = apps.get_model("ecommerce", "Product")
+    Product.objects.all().update(
+        is_private=models.Case(
+            models.When(visible_in_bulk_form=False, then=models.Value(True)),
+            default=models.Value(False),
+        )
+    )
+
+
+class Migration(migrations.Migration):
```

Comment:
I think we should be able to do this in a rename field migration, We should be able to rename the field name and then invert the values in a single migration. The operations would look something like this:

```
    operations = [
        migrations.RenameField(
            model_name='product',
            old_name='visible_in_bulk_form',
            new_name='is_private',
        )
    ]
```

Reply:
It still needed 3 steps, so, created a single file with 3 steps.